src/tracks/ui/PlayIndicatorOverlay.cpp

```diff
@@ -62,10 +62,8 @@ namespace {
 std::pair< wxPoint, wxBitmap > GetIndicatorBitmap( AudacityProject &project,
    wxCoord xx, bool playing)
 {
-   bool pinned = Scrubber::Get( project ).IsTransportingPinned();
-   wxBitmap & bmp = theTheme.Bitmap( pinned ?
-      (playing ? bmpPlayPointerPinned : bmpRecordPointerPinned) :
-      (playing ? bmpPlayPointer : bmpRecordPointer)
+   wxBitmap & bmp = theTheme.Bitmap(
+      playing ? bmpPlayPointer : bmpRecordPointer
```

Comment:
I see commit number 3 still colors the triangle on the timeline either red or green, even if the button is always green.